README.md

```diff
@@ -1,13 +1,15 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
+This week we started with our first backend project by creating an API using Express 💻
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## What I learned 🧠
 
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+- How to build an API in Node using Express
+- How to create routes in Express
+- Practice data manipulation in JavaScript - selecting, filtering, and limiting arrays
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Backend ➞ https://netflix-show.herokuapp.com/ 
+Front-end ➞ https://objective-hopper-5ee8ae.netlify.app/ 
+
```

Comment:
It would be very nice to have some instructions, how to use your API. Remember to make documentation on what endpoints and queries your user can use. 🤓